### DISCUSSION TITLE
HELP ME!! Camera internal parameter calculation

### DISCUSSION THREAD

**Discussion by huangmengjie** [cyberbotics/webots#6514] [Q&A]
How do webots calculate the relationship between the camera's image coordinates x, y and pixel coordinates u, v. How to calculate based on width heigth and fov

**Comment by ShuffleWire** [comment]
Does https://cyberbotics.com/doc/reference/camera help ? I can read 

vertical FOV = 2 * atan(tan(fieldOfView * 0.5) * (height / width))

and then https://cyberbotics.com/doc/reference/camera?version=master#spherical-and-cylindrical-projections could give infos
